### ISSUE TITLE
No validation on RHV RSA Key upload

### ISSUE THREAD

**Issue by kedark3** [ManageIQ/manageiq-providers-ovirt#266]
No validation on RHV RSA Key upload:
https://bugzilla.redhat.com/show_bug.cgi?id=1594343

**Comment by JPrause** [comment]
@kedark3 is this still a valid issue? If yes, let me know and I'll remove the stale label. If not can you close. If there's no update by next week, I'll be closing this issue.

**Comment by kedark3** [comment]
@JPrause Yes, it is still an issue. No validation. I can upload any file with any extension.

**Comment by JPrause** [comment]
Thanks @kedark3 I'll remove the stale label.
@miq-bot remove_label stale

**Comment by agrare** [comment]
cc @mwperina @borod108
